content/syllabuses/kotlin.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Android Development With Kotlin
+---
+##Week 1
+- Set - UP
+- {{% contentlink "topics/kotlin/set-up" %}}
+- Introduction to Kotlin
+- {{% contentlink "topics/kotlin/introduction" %}}
+- Basic Syntax & Types
+- {{% contentlink "topics/kotlin/basic-syntax-types" %}}
+- Basic - Control Flow
+- {{% contentlink "topics/kotlin/basic-control-flow" %}}
```

Comment:
Please remember that if you list absolutely evertying on the main page then every generated card will get highlighted.

What is a prerequisite to what? List the major outcomes on this page,

I'm thinking that the only thing that should live on this page is `- {{% contentlink "topics/kotlin/basic-control-flow" %}}`

Also it's important to look at the page that gets generated by hugo serve. Isn't this page amazingly repetitive?
Also it's important to look at the page that gets generated by hugo serve. Isn't this page amazingly repetitive?

Reply:
Also it's important to look at the page that gets generated by hugo serve. Isn't this page amazingly repetitive? 
I don't understand which page your referring to.

Reply:
I have made the other changes mentioned.